ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -259,8 +259,7 @@ objects:
           mkdir -p ~/.ssh
           cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_aws_rsa || true
           export PROVIDER_ARGS="-provider=aws -gce-zone=us-east-1"
-          # TODO: make openshift-tests auto-discover this from cluster config
```

Comment:
This is still a TODO.  The test suite shouldn't have to be told which region/zone to use, it should be able to discover that just by being pointed at a cluster with a kubeconfig.